skatepark/block-templates/page.html

```diff
@@ -8,7 +8,8 @@
 <main class="wp-block-group" style="padding-bottom:5em">
 
 <!-- wp:group {"align":"full"} -->
-<div class="wp-block-group alignfull"><!-- wp:post-featured-image {"align":"full"} /--></div>
+<div class="wp-block-group">
+<!-- wp:post-featured-image {"align":"full","style":{"color":{"duotone":["#000","#B9FB9C"]}}} /--></div>
```

Comment:
I assume we don't want this change?

Reply:
+1, we can ignore all the skatepark changes, that's done differently now